src/java.desktop/share/classes/sun/print/PSStreamPrintService.java

```diff
@@ -440,7 +450,7 @@ else if (attr.getCategory() == Copies.class) {
             if (flavor != null &&
                 !(flavor.equals(DocFlavor.SERVICE_FORMATTED.PAGEABLE) ||
                 flavor.equals(DocFlavor.SERVICE_FORMATTED.PRINTABLE))) {
-                return false;
+                return attr == SheetCollate.UNCOLLATED;
```

Comment:
Shouldn't we need to do the same for SheetCollate in WIn32PrintService#isATtributeValueSupported() where it always return false?